pegasus/sites.v3/code.org/public/athletes.haml

```diff
@@ -37,7 +37,7 @@ video_player: true
   - url = "https://www.youtube.com/watch?v=#{yt_code}"
   - facebook = {u: url}
   - twitter = {url: url, related: 'codeorg', text: "#{I18n.t(:sports_og_description)} @codeorg"}
-  =view :display_video_thumbnail, id: 'athletes_video', video_code: yt_code, caption: I18n.t(:video_1min), play_button: 'center', facebook: facebook, twitter: twitter, download_path: download_link
```

Comment:
I know in that email thread Hadi wanted to shorten the caption to this but I think it's unnecessary to keep the length if we don't have a full caption. Makes more sense in context like the other video captions.